contact.php

```diff
@@ -0,0 +1,108 @@
+<?php
+$name = $_POST['name'];
+$phone = $_POST['phone'];
+$email = $_POST['email'];
+$dropdown = $_POST['dropdown'];
+$message = $_POST['message'];
+ 
+$strValidationMessage = "";
+ 
+$to = "example@rawad.com";
+$subject = "New Message";
+ 
+if (isset($_POST['submit']))
+{
+  $name = $_POST['name'];
+  $phone = $_POST['phone'];
+  $email = $_POST['email'];
+  $message = $_POST['message'];
+ 
+  $boolValidateOK = 1;
```

Comment:
You are using this as a flag but you also have $strValidationMessage that you could check the contents of. I would lose the flag. I often times do something like this:

```
$errors = array();
```

```
if (/* email validation fails */) {
    $errors[] = "A valid email address is required.";
}
if (/* phone validation fails */) {
    $errors[] = "A valid phone number is required.";
}
```

```
if (empty($errors)) {
    // send mail
} else {
    foreach ($errors as $error) {
        echo "\n" . $error . "<br />";
    }
}
```